app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+<%= post.each do |post| %>
```

Comment:
- [ ] When looping, you don't need the `erb` tags that render. You just need the normal `erb` tags. If you use the `erb` tags that render, you will display the result of the query as seen in the image below:

![image](https://user-images.githubusercontent.com/77106962/225449771-a56d7847-345a-4002-bb55-98bd4ea49da3.png)

Kindly adjust it to `<% post.each do |post| %>`